apis/r/src/rinterface.cpp

```diff
@@ -434,6 +434,55 @@ bool c_allows_dups(const std::string& uri, Rcpp::XPtr<somactx_wrap_t> ctxxp) {
     return sch->allows_dups();
 }
 
+// [[Rcpp::export]]
+double c_capacity(const std::string& uri, Rcpp::XPtr<somactx_wrap_t> ctxxp) {
+    auto sr = tdbs::SOMAArray::open(OpenMode::read, uri, ctxxp->ctxptr);
+    std::shared_ptr<tiledb::ArraySchema> sch = sr->tiledb_schema();
+    sr->close();
+
+    uint64_t cap = sch->capacity();
+    return static_cast<double>(cap);
+}
+
+// Taken from tiledb-r
```

Comment:
Maybe move to `apis/r/src/rutilities.cpp`

Reply:
Moved in 2c35d5a